doc/Developing.md

```diff
@@ -148,6 +152,35 @@ To build and run the electron example:
 By default this will generate the code coverage for the tests in a HTML format, which can be easily viewed with your browser (chrome/firefox/edge/safari etc.) by opening `index.html`
 
 
+## Building on Windows
+
+Run cmd.exe as an administrator and install `choco` by copy-pasting the command to your console:
+```
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Install `yarn` via `choco`. The `yarn` installation ensures that you will have Node.js and npm too:
+```
+choco install yarn
+```
+
+Install `git` via `choco`.
```

Comment:
Please do not get into `git`. It is out of the scope of Theia. Could you please remove it?